nimble/controller/include/controller/ble_ll.h

```diff
@@ -199,44 +199,52 @@ extern STATS_SECT_DECL(ble_ll_stats) ble_ll_stats;
-#define BLE_LL_FEAT_LE_POWER_CLASS_1 (0x00008000)
-#define BLE_LL_FEAT_MIN_USED_CHAN    (0x00010000)
-#define BLE_LL_FEAT_CTE_REQ          (0x00020000)
-#define BLE_LL_FEAT_CTE_RSP          (0x00040000)
-#define BLE_LL_FEAT_CTE_TX           (0x00080000)
-#define BLE_LL_FEAT_CTE_RX           (0x00100000)
-#define BLE_LL_FEAT_CTE_AOD          (0x00200000)
-#define BLE_LL_FEAT_CTE_AOA          (0x00400000)
-#define BLE_LL_FEAT_CTE_RECV         (0x00800000)
-#define BLE_LL_FEAT_SYNC_SEND        (0x01000000)
-#define BLE_LL_FEAT_SYNC_RECV        (0x02000000)
-#define BLE_LL_FEAT_SCA_UPDATE       (0x04000000)
-#define BLE_LL_FEAT_REM_PKEY         (0x08000000)
+#define BLE_LL_FEAT_LE_ENCRYPTION    (0x0000000001)
+#define BLE_LL_FEAT_CONN_PARM_REQ    (0x0000000002)
+#define BLE_LL_FEAT_EXTENDED_REJ     (0x0000000004)
+#define BLE_LL_FEAT_SLAVE_INIT       (0x0000000008)
+#define BLE_LL_FEAT_LE_PING          (0x0000000010)
+#define BLE_LL_FEAT_DATA_LEN_EXT     (0x0000000020)
+#define BLE_LL_FEAT_LL_PRIVACY       (0x0000000040)
+#define BLE_LL_FEAT_EXT_SCAN_FILT    (0x0000000080)
+#define BLE_LL_FEAT_LE_2M_PHY        (0x0000000100)
+#define BLE_LL_FEAT_STABLE_MOD_ID_TX (0x0000000200)
+#define BLE_LL_FEAT_STABLE_MOD_ID_RX (0x0000000400)
+#define BLE_LL_FEAT_LE_CODED_PHY     (0x0000000800)
+#define BLE_LL_FEAT_EXT_ADV          (0x0000001000)
+#define BLE_LL_FEAT_PERIODIC_ADV     (0x0000002000)
+#define BLE_LL_FEAT_CSA2             (0x0000004000)
+#define BLE_LL_FEAT_LE_POWER_CLASS_1 (0x0000008000)
+#define BLE_LL_FEAT_MIN_USED_CHAN    (0x0000010000)
+#define BLE_LL_FEAT_CTE_REQ          (0x0000020000)
+#define BLE_LL_FEAT_CTE_RSP          (0x0000040000)
+#define BLE_LL_FEAT_CTE_TX           (0x0000080000)
+#define BLE_LL_FEAT_CTE_RX           (0x0000100000)
+#define BLE_LL_FEAT_CTE_AOD          (0x0000200000)
+#define BLE_LL_FEAT_CTE_AOA          (0x0000400000)
+#define BLE_LL_FEAT_CTE_RECV         (0x0000800000)
+#define BLE_LL_FEAT_SYNC_SEND        (0x0001000000)
+#define BLE_LL_FEAT_SYNC_RECV        (0x0002000000)
+#define BLE_LL_FEAT_SCA_UPDATE       (0x0004000000)
+#define BLE_LL_FEAT_REM_PKEY         (0x0008000000)
+#define BLE_LL_FEAT_CTE_RX           (0x0000100000)
+#define BLE_LL_CIS_MASTER            (0x0010000000)
+#define BLE_LL_CIS_SLAVE             (0x0020000000)
+#define BLE_LL_ISO_BROADCASTER       (0x0040000000)
+#define BLE_LL_SYNCH_RECEIVER        (0x0080000000)
+#define BLE_LL_ISO_HOST_SUPPORT      (0x0100000000)
 
 /* This is initial mask, so if feature exchange will not happen,
  * but host will want to use this procedure, we will try. If not
  * succeed, feature bit will be cleared.
  * Look at LL Features above to find out what is allowed
  */
 #define BLE_LL_CONN_INITIAL_FEATURES    (0x00000022)
-
 #define BLE_LL_CONN_CLEAR_FEATURE(connsm, feature)   (connsm->conn_features &= ~(feature))
 
+/* All the features which can be controlled by the Host */
+#define BLE_LL_HOST_CONTROLLED_FEATURES (BLE_LL_ISO_HOST_SUPPORT)
```

Comment:
I think this should be 0 until some initial ISO implementation lands in

Reply:
This is as per spec. Where we say that our controller supports it or not is in different place: 
``/** Our supported features which can be controller by the host */
uint64_t g_ble_ll_supported_host_features = 0;``